src/components/mobile/mobile-pages/Forms.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <v-ons-page>
+
+    <v-ons-list>
+      <v-ons-list-header>Text input</v-ons-list-header>
+      <v-ons-list-item :modifier="$ons.platform.isAndroid() ? 'nodivider' : ''">
+        <div class="left">
+          <v-ons-icon icon="md-face" class="list-item__icon"></v-ons-icon>
+        </div>
+        <label class="center">
+          <v-ons-input float maxlength="20"
+            placeholder="Name"
+            v-model="name"
+          >
+          </v-ons-input>
+        </label>
+      </v-ons-list-item>
+      <v-ons-list-item :modifier="$ons.platform.isAndroid() ? 'nodivider' : ''">
+        <div class="left">
+          <v-ons-icon icon="fa-question-circle-o" class="list-item__icon"></v-ons-icon>
+        </div>
+        <label class="center">
+          <v-ons-search-input maxlength="20"
+            placeholder="Search"
+            v-model="name"
+          >
+          </v-ons-search-input>
+        </label>
+      </v-ons-list-item>
+      <v-ons-list-item>
+        <div class="right right-label">
+          Hello {{ name || 'anonymous' }}!<v-ons-icon icon="fa-hand-spock-o" size="lg" class="right-icon"></v-ons-icon>
+        </div>
+      </v-ons-list-item>
+
+      <v-ons-list-header>Switches</v-ons-list-header>
+      <ons-list-item>
+        <label class="center" for="switch1">
+          Switch ({{ switchOn ? 'on' : 'off' }})
+        </label>
+        <div class="right">
+          <v-ons-switch input-id="switch1" v-model="switchOn">
+          </v-ons-switch>
+        </div>
+      </ons-list-item>
+      <ons-list-item>
+        <label class="center" for="switch2">
+          {{ switchOn ? 'Enabled switch' : 'Disabled switch' }}
+        </label>
+        <div class="right">
+          <v-ons-switch input-id="switch2" :disabled="!switchOn">
+          </v-ons-switch>
+        </div>
+      </ons-list-item>
+
+      <v-ons-list-header>Select</v-ons-list-header>
+      <ons-list-item>
+        <div class="center">
+          <v-ons-select style="width: 120px" v-model="selectedItem">
+            <option v-for="item in items" :value="item.value">
+              {{ item.text }}
+            </option>
+          </v-ons-select>
+
+        </div>
+        <div class="right right-label">
+          <s v-show="selectedItem !== 'Vue'">{{ selectedItem }}</s> Vue is awesome!
+        </div>
+      </ons-list-item>
+
+      <v-ons-list-header>Radio buttons</v-ons-list-header>
+      <v-ons-list-item v-for="(vegetable, $index) in vegetables" :key="vegetable"
```

Comment:
Are these vegetable examples, right?

Reply:
This page can be removed. It was present in the original demo.